cea/default.config

```diff
@@ -150,47 +152,58 @@ year.help = The year to base the calculations on. WARNING!, make sure you are us
 buildings =
 buildings.type = ListParameter
 buildings.help = a list of buildings to run the simulation for - leave blank to simulate all buildings
+buildings.category = Advanced
 
 loads-output =
 loads-output.type = MultiChoiceParameter
 loads-output.choices = QEf QHf QCf Ef Qhsf Qhs Qhsf_lat Qwwf Qww Qcsf Qcs Qcsf_lat Qcdataf Qcref Qhprof Edataf Ealf Eaf Elf Eref Eauxf Eauxf_ve Eauxf_hs Eauxf_cs Eauxf_ww Eauxf_fw Eprof Ecaf Egenf_cs
 loads-output.help = a list of loads to run the simulation for - leave blank to simulate all loads in demand_writers
+loads-output.category = Advanced
 
 massflows-output =
 massflows-output.type = MultiChoiceParameter
 massflows-output.choices = mcphsf mcpcsf mcpwwf mcpdataf mcpref
 massflows-output.help = a list of massflowrates to run the simulation for - leave blank to simulate all massflows in demand_writers
+massflows-output.category = Advanced
+
 
 temperatures-output =
 temperatures-output.type = MultiChoiceParameter
 temperatures-output.choices = Twwf_sup T_int Twwf_re Thsf_sup Thsf_re Tcsf_sup Tcsf_re Tcdataf_re Tcdataf_sup Tcref_re Tcref_sup
 temperatures-output.help = a list of temperatures to run the simulation for - leave blank to simulate all temperatures in demand_writers
+temperatures-output.category = Advanced
 
 resolution-output = hourly
 resolution-output.type = ChoiceParameter
 resolution-output.choices = hourly monthly
 resolution-output.help = resolution of output demand.
+resolution-output.category = Advanced
 
 format-output = csv
 format-output.type = ChoiceParameter
 format-output.choices = csv hdf5
 format-output.help = format of output files. csv is the default, HDF5 is used when data needs to be compressed.
+format-output.category = Advanced
 
 use-dynamic-infiltration-calculation = false
 use-dynamic-infiltration-calculation.type = BooleanParameter
 use-dynamic-infiltration-calculation.help = Use the dynamic infiltration calculation (slower!)
+use-dynamic-infiltration-calculation.category = Advanced
 
-use-daysim-radiation = true
+use-daysim-radiation = false
```

Comment:
this should be true

Reply:
you're right... i was setting this to false for veolia - i'll just have to also explain how to install daysim. no problem.